cocos/core/renderer/scene/model.ts

```diff
@@ -88,166 +88,387 @@ const lightmapSamplerWithMipHash = new SamplerInfo(
 );
 
 /**
- * A representation of a model
+ * @en A representation of a model instance
+ * @zh 代表一个模型实例
  */
 export class Model {
+    /**
+     * @en return sub models
+     * @zh 获取所有子模型
+     */
     get subModels () {
         return this._subModels;
     }
 
+    /**
+     * @en is inited
+     * @zh 是否初始化
+     */
     get inited (): boolean {
         return this._inited;
     }
 
+    /**
+     * @en return world aabb
+     * @zh 获取世界空间包围盒
+     */
     get worldBounds () {
         return this._worldBounds!;
     }
 
+    /**
+     * @en return model aabb
+     * @zh 获取模型包围盒
+     */
     get modelBounds () {
         return this._modelBounds;
     }
 
+    /**
+     * @en return model ubo buffer
+     * @zh 获取模型 ubo 缓冲
+     */
     get localBuffer () {
         return this._localBuffer;
     }
 
+    /**
+     * @en return world bound buffer
+     * @zh 获取世界包围盒 ubo 缓冲
+     */
     get worldBoundBuffer () {
         return this._worldBoundBuffer;
     }
 
+    /**
+     * @en return update time stamp
+     * @zh 获取更新时间戳
+     */
     get updateStamp () {
         return this._updateStamp;
     }
 
+    /**
+     * @en is instancing enabled
+     * @zh 是否开启实例化
+     */
     get isInstancingEnabled () {
         return this._instMatWorldIdx >= 0;
     }
 
+    /**
+     * @en return shadow bias
+     * @zh 获取阴影偏移值
+     */
```

Comment:
Don't write different API doc for getter and setter, they only need one API doc as they are considered as one property.